cinch/roles/jenkins_master/templates/roles_and_ldap_auth.groovy

```diff
@@ -0,0 +1,177 @@
+import hudson.*;
```

Comment:
There's some long lines in this file.  As of right now the standard is 80 characters (I know that this is not ideal in the view of many (myself included), but until the standard is officially changed we might as well adhere to it).